### ISSUE TITLE
Formaliser l'affichage des listes

### ISSUE THREAD

**Issue by htmfilho** [uclouvain/osis-documentation#52]
@glamarca commented on [Wed Apr 27 2016](https://github.com/uclouvain/osis/issues/501)

Liste initiale (avant recherche)
Liste vide (après recherche)
Liste pleine

---

@matrixise commented on [Fri Feb 17 2017](https://github.com/uclouvain/osis/issues/501#issuecomment-280611592)

Bonjour,

Quel est le niveau de cette issue ? Car elle ne bouge plus depuis juin 2016, 8 mois.

**Comment by basmot** [comment]
@glamarca @htmfilho Ce ticket a encore du sens?

**Comment by htmfilho** [comment]
Oui, parce que le comportement des écrans de recherche n'est pas le même par tout.
